lib/sass/rails/railtie.rb

```diff
@@ -32,7 +32,7 @@ class Railtie < ::Rails::Railtie
 
     # Remove the sass middleware if it gets inadvertently enabled by applications.
     config.after_initialize do |app|
```

Comment:
The proper fix is to change the middleware stack inside the initialization process. Can you change this block to a `initializer`?